src/utils/indent-normalizer.js

```diff
@@ -0,0 +1,18 @@
+// Indent to smallest non-empty whitespace level and trim start / end of string.
+const indentNormalizer = val => {
+  let prefix = null;
+  return (val || '')
+    .split('\n')
+    .filter(line => {
+      const [cur, remainder] = (line.match(/^([ ]*)([^ ]+)/) || []).slice(1);
+      return remainder
+        ? ((prefix =
+            null === prefix || cur.length < prefix.length ? cur : prefix),
+          !0)
+        : null !== prefix;
+    })
+    .map(line => (prefix ? line.replace(prefix, '') : line))
+    .join('\n')
+    .trimRight();
+};
+export default indentNormalizer;
```

Comment:
This is a function Ryan Roemer wrote for his own decks. If Markdown has too much indentation because of it's alignment in JSX (sometimes caused by Prettier) it will actually mutate the intention of the MD tags. This normalizes it and removes excess indentation and fixes those bugs.